Jenkinsfile

```diff
@@ -46,16 +59,29 @@ node ("ts-engine && heavy && java8") {
             build job: 'Nanoware/Omega/master', wait: false
         }
     }
-    stage('Analytics') {
-        sh "./gradlew --console=plain check spotbugsmain javadoc"
+
+    stage('Integration Tests') {
+        try {
+            sh './gradlew integrationTest'
+        } catch (e) {
+            echo 'Integration Tests failed'
+            throw e
+        } finally {
+            junit testResults: '**build/test-restults/integrationTest/*.xml', allowEmptyResults: true
```

Comment:
In Python, we can do a try/finally without defining a catch at all. Since we're just re-throwing the error, can we do the same in groovy?

Reply:
let's try that out :slightly_smiling_face: